README.md

```diff
@@ -53,4 +53,21 @@ Once the container is running, it should be accessible using:
 $ curl 127.0.0.1:5000
 ```
 
-Personally, I use a Mac to generate my code and docker image so I just open a browser to http://192.168.99.100:5000/`
+# Openshift Usage
+
+create the s2i builder image, dont need service or deployment
+
+    oc new-project aspnet --display-name="ASP.NET s2i" --description='ASP.NET s2i'
+    oc new-app https://github.com/eformat/s2i-aspnet --name=aspapp --strategy=docker
+    oc delete svc aspapp
+    oc delete dc aspapp
+
+s2i needs access to docker dameon, so as the cluster amdin
```

Comment:
you have a typo for "admin" and I also don't think the privileged scc is required now that we are running with `USER 1001` in the Dockerfile. @alberttwong can you confirm?